### PR TITLE
Nse 10515 GitHub actions

### PR DESCRIPTION
**Pull request by visalachundru**
Please review the github actions for errored-datums-reader and processed-datums-reader pipelines

### CODE REVIEW

File: .github/workflow_pending/CERT_errored_datums_reader.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          done
+          echo "Full changed file list: $file_list"
+          echo "changed_file_list=$file_list" >> "$GITHUB_OUTPUT"
+
+  update-pipelines:
+    needs: changed_files
+    runs-on: arc-neon-gke
+    #runs-on: ubuntu-latest 
+    env:
+      PACHD_ADDRESS: grpcs://pachyderm-prod.transitions.gcp.neoninternal.org:443
```
Comment (covesturtevant) [inline .github/workflow_pending/CERT_errored_datums_reader.yml:58]: Needs to use the cert version for both the PACHD_ADDRESS and PACH_TOKEN

**Review by covesturtevant — CHANGES_REQUESTED** [review]
The test of the github action for DEV-errored-datums-reader actually ran the pipeline update script for li191r-parQuantumLine DAG. See the logs. If it were pointing to the proper files it would have failed because edits are needed for it to run properly. 

Changes needed:
- Both the modules/processed_datums_reader and the modules/errored_datums_reader directories need build_tag_push_update.sh scripts that perform the docker building and tagging process for these modules. Use the script for any other module in the modules/ folder as a template. Run the script after committing the changes to generate the images tagged with the commit SHA.
- The pipeline yamls for the processed_datums_reader and the errored_datums_reader need to be in their own folders in the pipe/ directory and be accompanied by a pipe_list file. This file is used by the the github actions script to update the pipelines. Use the pipe_list file found in pipe/li191r as a template and modify accordingly.
- The github actions need to specify the PATHS environment variable pointing to the folder(s) with the pipeline file(s) in it.
- Ensure that the proper pachyderm environment and access token are specified in each the DEV, CERT, and PROD actions.

**Comment by covesturtevant** [comment]
Also, the level1_pipeline_list file should be in Github and a Github action created to updated it.

File: .github/workflows/CERT_errored_datums_reader.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          echo "Full changed file list: $file_list"
+          echo "changed_file_list=$file_list" >> "$GITHUB_OUTPUT"
+
+  update-pipelines:
+    needs: changed_files
+    runs-on: arc-neon-gke
+    #runs-on: ubuntu-latest 
+    env:
+      PACHD_ADDRESS: grpcs://pachyderm-prod.transitions.gcp.neoninternal.org:443
+      PACH_TOKEN: ${{ secrets.RepoOwnerPachydermProd }}
```
Comment (covesturtevant) [inline .github/workflows/CERT_errored_datums_reader.yml:61]: @visalachundru  address and token need to point to cert for the cert workflow

Comment (visalachundru) [reply .github/workflows/CERT_errored_datums_reader.yml]: Cove, please review the changes that I made to this branch, also please review the "PROD-level1-pipeline-list"  git hub actions files. I'm not sure about the  "OUT_PATHS" in the "env" section. I'm not sure if I have to give the whole pipeline path or not. 
I would like to test in cert.

**Comment by covesturtevant** [comment]
Looks good @visalachundru! I made a few minor edits. I'll merge to cert as well so you can test there.

**Review by covesturtevant — APPROVED** [review]

**Pull request merged**